### PR TITLE
OTB Long-term care flow

### PR DESCRIPTION
**Pull request by brdunfield**
https://trello.com/c/IOGk9BZy/464-m-update-otb-long-term-care-flow

This PR spawns a couple associated issues that I've decided to split out into separate cards since they're a bit outside the scope of the original card.

Firstly, this PR introduces a new use-case which isn't covered by the current doYesNo middleware. I've created a card to address this issue here: https://trello.com/c/qzJUdDs8/516-m-refactor-doyesno-function-to-support-branching-decisions

Additionally, at this point deductions.spec is a bit of a mess from integrating all the new OTB flow pages / tests. Once all the OTB pages have been merged, I've created a card to clean up this file. More details here: https://trello.com/c/aTJDBuy6/515-s-refactor-tests-around-otb-flow

This flow is a bit more complex than some of the others that have gone in for OTB so far - more pages were added, and there's the new branching amounts flow (either the full amount or the room and board amount).

### CODE REVIEW

File: locales/en.json [hunk 1]
```diff
@@ -1,4 +1,7 @@
 {
+  "45": "45",
+  "50": "50",
+  "4566": "4566",
```
Comment (pcraig3) [inline locales/en.json]: Can we delete these numbers?

File: views/deductions/trillium-energy-cost.pug [hunk 2]
```diff
@@ -11,6 +11,8 @@ block content
   form.cra-form(method='post')
     +radiosYesNo('trilliumEnergyCostClaim', 'In 2018, when you lived on an Ontario reserve, did you pay home energy costs, for example, to heat or cool your home, or use appliances like a stove or fridge?', trilliumEnergyCostClaim)
 
+    include ../_includes/principalResidence
```
Comment (pcraig3) [inline views/deductions/trillium-energy-cost.pug]: Was this left out of a previous PR?

File: routes/deductions/deductions.controller.js [hunk 3]
```diff
@@ -117,15 +120,52 @@ module.exports = function(app) {
...
+      'trilliumLongTermCareAmount',
+    ]),
+    doRedirect,
+  )
+
   app.get('/trillium/longTermCare/type', renderWithData('deductions/trillium-longTermCare-type'))
   app.post(
     '/trillium/longTermCare/type',
-    checkSchema(trilliumLongTermCareTypeSchema),
+    checkSchema(trilliumlongTermCareTypeSchema),
```
Comment (pcraig3) [inline routes/deductions/deductions.controller.js]: Weird that this shows up as an addition/deletion because these lines are identical, right?

Comment (brdunfield) [reply routes/deductions/deductions.controller.js]: The casing changed - there was some inconsistency with Long being spelled with a capital or lowercase. I made this one the same as the other schema.

File: views/deductions/trillium-energy-cost.pug (hunk 2 above)
Comment (brdunfield) [reply views/deductions/trillium-energy-cost.pug]: Ah, yes. I forgot to note it in my comment as I noticed it the other day and it slipped my mind today. It is supposed to be on this page per the content doc, but got missed in the PR for that page.

File: locales/en.json (hunk 1 above)
Comment (brdunfield) [reply locales/en.json]: Oops. I'll have to clean this up in a new PR. Maybe I'll bundle it in with the test cleanup issue?

**Pull request merged**